### PR TITLE
WL - Add "play round" functionality into existing structure. document structure in README

### PR DESCRIPTION
**Pull request by weslau**

### CODE REVIEW

File: app/gameplay.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    ## HOW TO WIN
+
+    In a 3, 4, or 6-player game, the player with the most points after 12 rounds wins the game. In a 5-player game, the player with the most points after 10 rounds wins the game. If there is a tie at the end, continue play until the tie is broken and a winner is declared. For a quicker game, all players can simply have one less round playing The Victim.
+
+    ## NOTES ON PLAY
+
+    - The Victim changes every round, and every player has an equal number of rounds playing The Victim.
+    - Always rank the cards secretly based on how you think The Victim will rank the cards.
+    ''')
```
Comment (matthewncover) [inline app/gameplay.py:108]: will need to adjust some of the language to suit a webapp and not a physical deck of cards

**Review by matthewncover — APPROVED** [review]
lgtm

**Review by michaelmusun — APPROVED** [review]

File: app/main.py [hunk 2]
```diff
@@ -72,8 +162,9 @@ def game_start_page():
         "create account": create_account_page,
         "login existing account": existing_account_page,
         "welcome": welcome_page,
-        "lobby": lobby_page,
-        "game start": game_start_page
+        "lobby": lobby_page, ## can we combine lobby page and game_start_page together?
```
[the remark this answers is not in the document]
Comment (weslau) [reply app/main.py]: done

File: app/gameplay.py (hunk 1 above)
Comment (weslau) [reply app/gameplay.py:107]: changed language

**Pull request merged**